Add a note regarding the Datadog Agent service account (#10928)
DataDog/documentation: content/en/agent/faq/windows-agent-ddagent-user.md

* It has remote login disabled
* It has network login disabled

> **Note**: The installer doesn't add the account it creates to the `Users` groups by default. On most systems this is fine.

Reviewer: ```suggestion
**Note**: The installer doesn't add the account it creates to the `Users` groups by default.
```

Reviewer: @julien-lebot Suggestion updated

> If you encounter permission issues running the Datadog Agent service, make sure the account under which the service runs is in the `Users` group.

Reviewer: ```suggestion
If you encounter permission issues running the Datadog Agent service, add the account the service runs on to the `Users` group.
```
